### ISSUE TITLE
redirect broken with meteor beta 11

### ISSUE THREAD

**Issue by timothyarmes** [kadirahq/flow-router#539]
Hi,

Using the latest versions of flow-router-ssr and meteor beta 11 my app now crashes when I try to redirect from inside a trigger.

Tracing the code, the bug appears to be a Babel issue. I've reported it here:

https://github.com/meteor/meteor/issues/6312

I'm also opening an issue here so that it can be tracked.

**Comment by arunoda** [comment]
Thanks.

**Comment by arunoda** [comment]
Seems like this will be fixed in beta-12.

**Comment by dufia** [comment]
Hey @arunoda 

Is redirecting broken? I use beta-12 and I get:
  ```
Uncaught TypeError: str.slice is not a function
 ```

 ```
while ((res = PATH_REGEXP.exec(str)) != null) {
    var m = res[0]
    var escaped = res[1]
    var offset = res.index
    path += str.slice(index, offset)
    index = offset + m.length
```

Thanks

**Comment by dufia** [comment]
@timothyarmes It seems to be fixed in release/METEOR@1.3-rc.8 that came out two days ago

**Comment by timothyarmes** [comment]
Yes, was fixed in rc1- I'll close this.
